README.md

```diff
@@ -44,5 +44,5 @@ This is the internal repo to for Project Management of the IPFS GUI project.
   - [Design Tools Research](https://docs.google.com/document/d/1qJyfwgcMg8l3Tk3aYxF38iyYRhkEf3nlLNqOw4ZiW_8/edit)
 - Code
   - [webui](https://github.com/ipfs/webui)
```

Comment:
While you are at it, wanna fix also this link to https://github.com/ipfs-shipyard/ipfs-webui ?

Reply:
I went ahead and did it :)